src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -224,13 +225,22 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Supported create container/blob Sas token with new permission x,t
-    -  ''New-AzStorageBlobSASToken''
-    -  ''New-AzStorageContainerSASToken''
-* Supported create account Sas token with new permission x,t,f
-    -  ''New-AzStorageAccountSASToken''
-* Supported get single file share usage
-    - ''Get-AzRmStorageShare'''
+        ReleaseNotes = '* Support set/update/remove Acl recursively on Datalake Gen2 item
+    -  ''Set-AzDataLakeGen2AclRecursive''
+    -  ''Update-AzDataLakeGen2AclRecursive''
+    -  ''Remove-AzDataLakeGen2AclRecursive''
+* Supported Container access policy with new permission x,t
```

Comment:
Seems we will not release the container access policy related changes, so we should not add them to change log:

```
* Supported Container access policy with new permission x,t
    -  ''New-AzStorageContainerStoredAccessPolicy''
    -  ''Set-AzStorageContainerStoredAccessPolicy''
* Change the output of get/set Container/Share/Queue/Table access policy cmdlet, by change the child property Permission type from enum to String
    -  ''Get-AzStorageContainerStoredAccessPolicy''
    -  ''Set-AzStorageContainerStoredAccessPolicy''
    -  ''Get-AzStorageShareStoredAccessPolicy''
    -  ''Set-AzStorageShareStoredAccessPolicy''
    -  ''Get-AzStorageQueueStoredAccessPolicy''
    -  ''Set-AzStorageQueueStoredAccessPolicy''
    -  ''Get-AzStorageTableStoredAccessPolicy''
    -  ''Set-AzStorageTableStoredAccessPolicy'''
```